openpype/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -39,8 +39,7 @@ def process(self, instance):
         start = float(instance.data.get("frameStartHandle", 1))
         end = float(instance.data.get("frameEndHandle", 1))
 
-        attrs = instance.data.get("attr", "").split(";")
-        attrs = [value for value in attrs if value.strip()]
+        attrs = instance.data.get("userDefinedAttributes", [])
```

Comment:
This is breaking the support of the `attr` setting that is on the instances. Functionality should be preserved.

Reply:
Yup good point.